Maquetado/scss/details.scss

```diff
@@ -0,0 +1,65 @@
+.book {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-transparent;
+  display: flex;
+  flex-flow: column wrap;
+  height: 100%;
+  margin: 0 auto;
+  max-height: 415px;
+  max-width: 1000px;
+  width: 100%;
+}
+
+.book-info {
+  margin: 10px auto;
+}
+
+.book-header {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 10px;
+}
+
+.book-item {
```

Comment:
This properties are the same that `book-header` the only difference is `margin`. What do you think about of use `@extend` or selectors?